components/Sidebar/UserProfileButton.tsx

```diff
@@ -49,29 +38,7 @@ const UserProfileButton = () => {
         </p>
         <p className="text-xs text-muted-foreground truncate">{organization}</p>
       </div>
-      <DropdownMenu>
-        <DropdownMenuTrigger asChild>
-          <Button variant="ghost" className="p-0">
-            <ChevronDown className="h-4 w-4 shrink-0 text-muted-foreground ml-auto" />
-          </Button>
-        </DropdownMenuTrigger>
-        <DropdownMenuContent className="w-56" align="start">
-          <DropdownMenuLabel className="text-sm font-semibold">
-            My Account
-          </DropdownMenuLabel>
-          <DropdownMenuGroup>
-            <DropdownMenuItem onClick={toggleModal} className="cursor-pointer">
-              <IconUser />
-              Profile
-            </DropdownMenuItem>
-          </DropdownMenuGroup>
-          <DropdownMenuSeparator />
-          <DropdownMenuItem className="cursor-pointer" onClick={signOut}>
-            <IconLogout />
-            Log out
-          </DropdownMenuItem>
-        </DropdownMenuContent>
-      </DropdownMenu>
+      <UserProfileDropdown onProfileClick={toggleModal} onSignOut={signOut} />
```

Comment:
Why pass these as props?
- Wouldn't the code be simpler if you just imported both of these from the `useUserProvider` directly within `UserProfileDropdown`?